examples/compute-cost/program_logs/EvQ2hf9K3C6UuoMoseaGdrAiFxqd6AGYBacXrfwB8qv5.compute_cost.log

```diff
@@ -0,0 +1,242 @@
+Streaming transaction logs mentioning EvQ2hf9K3C6UuoMoseaGdrAiFxqd6AGYBacXrfwB8qv5. Confirmed commitment
```

Comment:
are these logs not gitignored? @armaniferrante

Reply:
I'm going to remove all the commits unrelated to the mint changes